### PR TITLE
Illustrator Test Scripts Updated.

### PR DESCRIPTION
**Pull request by PavanKumarN8**
Hi, @sigadamvenkata, @JackySun9, @Dli3, and @skumar09, I have updated the Test script as discussed.

Locale Test Run Results
image

### CODE REVIEW

**Review by skumar09 — APPROVED** [review]
Looks good to me

File: selectors/feds/feds.illustratorsanity.page.js [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+          if (country === 'Poland') {
+            await expect(element).toBeVisible();
+          }
+          break;
+        default:
+          await expect(element).toBeVisible();
+      }
+    }));
+
+    await this.creativityAndDesign.click();
```
Comment (Dli3) [inline selectors/feds/feds.illustratorsanity.page.js:148]: Why do you need this click here? It looks like your test steps are just calling 
`    // Verifying the visibility of U-NAV Elements
    await illustrate.validatingunavElements();
    // Verifying the Visibility of Creativity & Design Elements
    await illustrate.validatingCreativityAndDesignElementsSecondSet();
    // Verifying the visibility of Footer Elements
    await illustrate.validatingfooterElements();`
Where each function starts with `    await this.creativityAndDesign.click();
` and ends with `    await this.creativityAndDesign.click();
`.

File: selectors/feds/feds.illustratorsanity.page.js [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+  }
+
+  async validatingCreativityAndDesignElementsSecondSet() {
```
Comment (Dli3) [inline selectors/feds/feds.illustratorsanity.page.js:151]: Looks like these functions are doing similar actions. You can probably combine them to keep it dry and just pass in the list of elements for each set to the function.

File: tests/feds/illustratorSanity.test.js [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+
+  beforeEach(async ({ page }) => {
+    illustrate = new IllustratorPageSanity(page);
+  });
+
+  afterEach(async ({ page }) => {
+    await page.close();
+  });
+
+  test(`${features[0].name}, ${features[0].tags}, ${features[0].country}`, async ({ page, baseURL }) => {
```
[the remark this answers is not in the document]
Comment (Dli3) [reply tests/feds/illustratorSanity.test.js:16]: Agreed @JackySun9 . There's a lot of test cases that repeat.

File: tests/feds/illustratorSanity.test.js [hunk 4]
```diff
@@ -0,0 +1,362 @@
...
+
+  test(`${features[21].name}, ${features[21].tags}, ${features[21].country}`, async ({ page, baseURL }) => {
+    console.info(`[FEDSInfo] Checking Page: ${baseURL}${features[21].path}`);
+
+    const pageURL = `${baseURL}${features[21].path}`;
+    await page.goto(pageURL, { waitUntil: 'domcontentloaded' });
+    await expect(page).toHaveURL(pageURL);
+
+    // Verifying the visibility of U-NAV Elements
+    await illustrate.validatingunavElementsSecondSet();
```
Comment (skumar09) [inline tests/feds/illustratorSanity.test.js:341]: small update: Pls camel case method names correct, 

- `validatingUnavEleementsSecondSet()`
- `validatingFooterElements`

File: tests/feds/illustratorSanity.test.js [hunk 5]
```diff
@@ -0,0 +1,362 @@
...
+    // Verifying the visibility of U-NAV Elements
+    await illustrate.validatingunavElements();
+    // Verifying the Visibility of Creativity & Design Elements
+    await illustrate.validatingCreativityAndDesignElementsSecondSet();
+    // Verifying the visibility of Footer Elements
+    await illustrate.validatingfooterElements();
+  });
+
+  test(`${features[4].name}, ${features[4].tags}, ${features[4].country}`, async ({ page, baseURL }) => {
+    // const illustrate = new IllustratorPageSanity(page);
```
Comment (Dli3) [inline tests/feds/illustratorSanity.test.js:77]: Do you need this commented out line still?

File: tests/feds/illustratorSanity.test.js [hunk 6]
```diff
@@ -0,0 +1,362 @@
...
+    // Verifying the Visibility of Creativity & Design Elements
+    await illustrate.validatingCreativityAndDesignElements();
+    // Verifying the visibility of Footer Elements
+    await illustrate.validatingfooterElements();
+  });
+
+  test(`${features[5].name}, ${features[5].tags}, ${features[5].country}`, async ({ page, baseURL }) => {
+    console.info(`[FEDSInfo] Checking Page: ${baseURL}${features[5].path}`);
+
+    // await test.step(`Validating ${features[5].country} Page`, async () => {
```
Comment (Dli3) [inline tests/feds/illustratorSanity.test.js:95]: Do you need this commented out line still?

**Review by sigadamvenkata — APPROVED** [review]

**Pull request merged**